### ISSUE TITLE
Windows development

### ISSUE THREAD

**Issue by jfly** [cubing/worldcubeassociation.org#11]
I worked pretty hard to make sure that stuff works in both Windows and Linux, but unfortunately, I had to resort to symlinks for the rails `public/assets`, `public/files`, and `public/images` directories.

We may be able to get things working on Windows by falling back to having rails serve up static content: https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/config/environments/production.rb#L25.

**Comment by lgarron** [comment]
Should the comment in the README be updated?

**Comment by jfly** [comment]
This issue should be reopened, unfortunately. I moved the image upload folders into the secrets/ directory. I have been trying to avoid touching the old PHP code as much as possible, so I opted to make a symlink for the uploads directory here: https://github.com/cubing/wca-website-php/blob/master/upload. This symlink can be removed by editing the PHP code that interacts with the upload directory to instead look at `../../secrets/upload`. I'll reopen this issue to track fixing that, and update the README accordingly.
